app/views/govuk_publishing_components/components/_image_card.html.erb

```diff
@@ -44,9 +44,9 @@
         <%= card_helper.context %>
       </div>
       <%= card_helper.description %>
-      <% if card_helper.extra_links.any? %>
-        <ul class="gem-c-image-card__list <%= "gem-c-image-card__list--indented" if not card_helper.extra_links_no_indent %>">
-          <% card_helper.extra_links.each do |link| %>
+      <% if card_helper.extra_list.any? %>
+        <ul class="gem-c-image-card__list <%= "gem-c-image-card__list--indented" if not card_helper.extra_list_no_indent %>">
+          <% card_helper.extra_list.each do |link| %>
```

Comment:
This was originally called `extra_links` because these were links in addition to the main link of the image itself. I see why you're renaming this, although `extra_list` to my mind implies that this is a list in addition to another list - but this is the only list. Would it make more sense to call it something like `image_list` or `extra_details` or even just `list`?

I realise I'm suggesting this a bit late in the day, so totally non-blocking.

Reply:
I think I'm keen to see this change, tbh. I'm in favour of clarity of language.

Reply:
> `extra_list` to my mind implies that this is a list in addition to another list

🤔 Fair - I've gone within `extra_details` in this context, `image_list` made me think there would be more images and just `list` sort of made me think there wouldn't be links in the default - which there are.